ambry-router/src/main/java/com.github.ambry.router/PutManager.java

```diff
@@ -64,6 +64,7 @@
   private final AtomicBoolean isOpen = new AtomicBoolean(true);
   private final OperationCompleteCallback operationCompleteCallback;
   private final ReadyForPollCallback readyForPollCallback;
+  private final List<String> listToFillWithIdsToDelete;
```

Comment:
just thought that `idsToDelete` had the connotation of these being a list with ids passed to the put manager for it to delete. If you guys did not feel so, I think I will change it back.